src/libs/API/parameters/UpdatePolicyConnectionConfigParams.ts

```diff
@@ -4,7 +4,7 @@ type UpdatePolicyConnectionConfigParams<TConnectionName extends ConnectionName,
     policyID: string;
     connectionName: TConnectionName;
     settingName: TSettingName;
-    settingValue: Connections[TConnectionName]['config'][TSettingName];
+    settingValue: Connections[TConnectionName]['config'][TSettingName] | null;
```

Comment:
We can't send null. The formData will send that as a string. Ref: https://developer.mozilla.org/en-US/docs/Web/API/FormData/set#value

Reply:
@s77rt we need to reset value - and make it either null or undefined? should we send undefined instead of null?

Reply:
Yes, `undefined` will not send that field. The BE should pick this up as we are trying to unset the value. That's how `updateGroupChatAvatar` works.

Reply:
@s77rt Which mean that we should just send anything? like empty string?